src/bin/cql-stress-cassandra-stress/settings/command/help.rs

```diff
@@ -40,5 +46,6 @@ pub fn parse_help_command(payload: &mut ParsePayload) -> Result<CommandParams> {
         return print_command_help(params[0]);
     }
 
-    todo!("Implement help for options.");
+    // Print help for option.
+    print_option_help(payload.iter().next().unwrap().0)
```

Comment:
> The one `unwrap()` that still begins the function is upsetting. Please consider the possible ways to avoid it (my quick research hints that it can be too expensive), considering the caller's logic)

That's true. The fact is that the caller of this function ensures that there will be a `"help"` key in the map (in `ParsePayload`, which maps command/option to their corresponding parameters). However, I still think that we could return an empty vector here for two reasons:

- we get rid of panic
- an empty vector is properly handled later in the function

Reply:
> In Rust, we strive to avoid `unwrap()`s where possible. Static, syntactic guarantees are preferred (`match` and `if let` come in handy for these). To see why, keep reading.

Thank you for the thorough explanation! I definitely see how we benefit from the changes you introduced.